source/Hit:Stand.py

```diff
@@ -0,0 +1,17 @@
+def Hit_Stand(self, Player):
```

Comment:
self is used for classes. Hit_Stand is a function so it should just be 
def Hit_Stand(Player):

Reply:
noted